src/components/Image/Image.tsx

```diff
@@ -17,10 +17,14 @@ export interface ImageProps extends TPAComponentProps {
   onLoad?: React.EventHandler<React.SyntheticEvent>;
   /** A callback to be called if error occurs while loading */
   onError?: React.EventHandler<React.SyntheticEvent>;
+  /** Specifies how the image is resized to fit its container */
+  resize?: 'contain' | 'cover';
```

Comment:
Actually we can rename this prop merely `cover`, since it's only two values and `contain` is the default anyway, but I think it's more clear like that and also there still has a potential we'll add more resize options in the future so I wouldn't do that to avoid breaking the API...